### ISSUE TITLE
myfinder托盘和常驻都无法打开SteelSeriesGG此软件

### ISSUE THREAD

**Issue by hufei110** [mydockfinder/mydockfinder#756]
其他软件能够正常在myfinder托盘和常驻内打开，只有SteelSeriesGG此软件无法正常打开，Windows托盘内能够正常打开，请问有什么解决方法吗？

**Comment by mydockfinder** [comment]
我适配一下，我记得之前适配过不过适配他以后其他图标就打不开了，我再想想别的办法




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月20日(星期天) 上午10:05
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; 
主题: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)














 
其他软件能够正常在myfinder托盘和常驻内打开，只有SteelSeriesGG此软件无法正常打开，Windows托盘内能够正常打开，请问有什么解决方法吗？

**Comment by hufei110** [comment]
好的 有劳了

**Comment by hufei110** [comment]
你好大大  请问 今天我发现myfinder 点击左上方的睡眠功能没反应，是我电脑的原因吗？我应该如何让他生效？

**Comment by hufei110** [comment]
是因为我是笔记本的原因吗？

**Comment by mydockfinder** [comment]
点完了显示器灭吗&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午4:02
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)














 
你好大大  请问 今天我发现myfinder 点击左上方的睡眠功能没反应，是我电脑的原因吗？我应该如何让他生效？

**Comment by hufei110** [comment]
点完过后，就跟没事发生一样

**Comment by mydockfinder** [comment]
没事&nbsp; 我适配了下个版本就好了，这个软件托盘激活和其他的软件不一样
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午4:03
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)














 
是因为我是笔记本的原因吗？

**Comment by hufei110** [comment]
好的

**Comment by mydockfinder** [comment]
这个还真不知道，我倒是知道开了休眠的话点睡眠也是休眠




№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mydockfinder/mydockfinder-for-Win10-Win11"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月21日(星期一) 下午4:05
***@***.***&gt;;
抄送:&nbsp;"№じ ***@***.******@***.***&gt;;
主题:&nbsp;Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)





 
点完过后，就跟没事发生一样

**Comment by hufei110** [comment]
好的  我自己先排查一下

**Comment by hufei110** [comment]
是的 我测试了一下  开启休眠模式，myfinder点睡眠，就直接成休眠了，我选择关闭它。 我在Windows里面选择睡眠，是可以正常睡眠的，我不知道是不是与睡眠状态有关，笔记本好像只有（s0）该状态睡眠。我不清楚这个猜测方向是否正确，是不是因为我的系统固件不支持该睡眠状态导致的。
![屏幕截图 2024-10-21 164120](https://github.com/user-attachments/assets/ab0eeb9e-f049-4cb6-95ae-c128cbe96a8b)

**Comment by mydockfinder** [comment]
有可能是这个原因，我找找例子有没有办法
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午4:49
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)














 
是的 我测试了一下  开启休眠模式，myfinder点睡眠，就直接成休眠了，我选择关闭它。 我在Windows里面选择睡眠，是可以正常睡眠的，我不知道是不是与睡眠状态有关，笔记本好像只有（s0）该状态睡眠。我不清楚这个猜测方向是否正确，是不是因为我的系统固件不支持该睡眠状态导致的。
 2024-10-21.164120.png (view on web)

**Comment by mydockfinder** [comment]
关机重启都正常吗




№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "mydockfinder/mydockfinder-for-Win10-Win11"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2024年10月21日(星期一) 下午4:49
***@***.***&gt;;
抄送:&nbsp;"№じ ***@***.******@***.***&gt;;
主题:&nbsp;Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)





 
是的 我测试了一下  开启休眠模式，myfinder点睡眠，就直接成休眠了，我选择关闭它。 我在Windows里面选择睡眠，是可以正常睡眠的，我不知道是不是与睡眠状态有关，笔记本好像只有（s0）该状态睡眠。我不清楚这个猜测方向是否正确，是不是因为我的系统固件不支持该睡眠状态导致的。
 2024-10-21.164120.png (view on web)

**Comment by hufei110** [comment]
是的  该栏其他选项一切正常

**Comment by mydockfinder** [comment]
好吧 给我整不会了




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午5:04
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)





 
是的  该栏其他选项一切正常
 
&mdash;

**Comment by mydockfinder** [comment]
打开CMD 然后输入&nbsp;&nbsp;rundll32.exe powrprof.dll,SetSuspendState 0,1,0&nbsp; &nbsp;回车看看能不能睡眠




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午5:04
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)





 
是的  该栏其他选项一切正常
 
&mdash;

**Comment by hufei110** [comment]
没有任何效果

**Comment by mydockfinder** [comment]
你别说 我刚才系统开关一次休眠后也不能睡眠了，没反应了。。。。。。&nbsp; &nbsp;我正好研究一下
&nbsp;


№じ&nbsp;尐乄鈊~→☆
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2024年10月21日(星期一) 下午5:16
收件人: ***@***.***&gt;; 
抄送: "№じ ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [mydockfinder/mydockfinder-for-Win10-Win11] myfinder托盘和常驻都无法打开SteelSeriesGG此软件 (Issue #756)














 
没有任何效果

**Comment by hufei110** [comment]
哈哈  好的
